extensions/dacpac/src/wizard/api/dacFxConfigPage.ts

```diff
@@ -182,7 +182,7 @@ export abstract class DacFxConfigPage extends BasePage {
 		if (this.fileTextBox.value && path.dirname(this.fileTextBox.value)) {
 			return path.dirname(this.fileTextBox.value);
 		} else { // otherwise use the folder open in the Explorer or the home directory
-			return vscode.workspace.workspaceFolders ? vscode.workspace.workspaceFolders[0].name : os.homedir();
+			return vscode.workspace.workspaceFolders ? vscode.workspace.workspaceFolders[0].uri.fsPath : os.homedir();
```

Comment:
why is this guy not using the util method?

Reply:
I only added the method in the schema compare extension's utils since it's only used once in the dacpac extension

Reply:
i see. you are good to go.